setup.cfg

```diff
@@ -27,6 +27,7 @@ classifiers =
 install_requires =
     requests~=2.28
     retry~=0.9.2
+    solr-to-es @ git+https://github.com/o19s/solr-to-es@master
```

Comment:
Replace `@master` with `@fc758840b07873de0fcaa2ccd675c79bcafc0b99` otherwise this is open to the [dependency confusion vulnerability](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610).

Reply:
Thanks @nutjob4life , will do